priv/perf/Makefile

```diff
@@ -16,7 +16,7 @@ init:
 	mix deps.get
 
 test: # runs test against child chain and geth provided in test docker containers
-	CONTRACT_ADDRESS_ETH_VAULT=0x4e3aeff70f022a6d4cc5947423887e7152826cf7 LOAD_TEST_FAUCET_PRIVATE_KEY=0xd885a307e35738f773d8c9c63c7a3f3977819274638d04aaf934a1e1158513ce CONTRACT_ADDRESS_PLASMA_FRAMEWORK=0xc673e4ffcb8464faff908a6804fe0e635af0ea2f mix test
+	CONTRACT_ADDRESS_ETH_VAULT=0x4e3aeff70f022a6d4cc5947423887e7152826cf7 LOAD_TEST_FAUCET_PRIVATE_KEY=0xd885a307e35738f773d8c9c63c7a3f3977819274638d04aaf934a1e1158513ce CONTRACT_ADDRESS_PLASMA_FRAMEWORK=0xc673e4ffcb8464faff908a6804fe0e635af0ea2f CONTRACT_ADDRESS_PAYMENT_EXIT_GAME=0x89afce326e7da55647d22e24336c6a2816c99f6b mix test --trace
```

Comment:
why is this data hardcoded?

Reply:
should I use `localchain_contract_addresses.env` file?

Reply:
yeah, that's like the location for all contract addresses coming from the snapshot, everyone should be using this, so that when we upgrade to v2 contracts, we change only the snapshot URL.